packages/timeline-state-resolver/src/integrations/vizMSE/index.ts

```diff
@@ -346,6 +362,50 @@ export class VizMSEDevice extends DeviceWithState<VizMSEState, DeviceOptionsVizM
 		return state
 	}
 
+	private _content2StateLayer(
```

Comment:
Why not just call it `_contentToStateLayer`? 
Would be nice if we could resemble english as much as possible :)

Reply:
Ohh, I see it was a function already there.
In that case, boyscouting the name would be nice :)